src/dmd/typinf.d

```diff
@@ -29,9 +29,16 @@ import dmd.visitor;
 extern (C++) void genTypeInfo(Type torig, Scope* sc)
 {
     //printf("Type::genTypeInfo() %p, %s\n", this, toChars());
+
+    if (!global.params.useTypeInfo)
```

Comment:
This block of code was modified to make it consistent with the following in statementsem.d

https://github.com/dlang/dmd/blob/ec4534ec63958b17167d636cd8b98824ce52f7d3/src/dmd/statementsem.d#L3626-L3636